spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/metrics/export/otlp/OtlpProperties.java

```diff
@@ -79,6 +79,11 @@ public class OtlpProperties extends StepRegistryProperties {
 	 */
 	private TimeUnit baseTimeUnit = TimeUnit.MILLISECONDS;
 
+	/**
+	 * Whether virtual threads should be used for publishing metrics.
+	 */
+	private boolean virtualThreadsEnabled = false;
```

Comment:
We don't need a new property here. We have a condition, named `ConditionalOnThreading` which can be used to detect if the user wants to enable virtual threads. There are already usages in the codebase for this annotation which you can use to see how it's used.

Reply:
Great, thanks for sharing that. I'll switch to `ConditionalOnThreading`.